recipe/meta.yaml

```diff
@@ -10,32 +10,41 @@ source:
   sha256: 2770c4928abc721f46b705e6a72b0c59480c4a69c9a83ca0b00bb994f1ea4b32
 
 build:
-  noarch: python
   number: 0
   script: cd {{ RECIPE_DIR }} && {{ PYTHON }} builder.py
+  skip: True  # [py<37]
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - typing_extensions
 
 test:
   requires:
     - pip
-  imports:
-    - pyee
+    - pkgconfig
+    - pkg-config
+    - python
```

Comment:
These packages are redundant in test/requires:
```
    - pkgconfig
    - pkg-config
    - python
```
`run_test.py` requires only `pkg_resources` (I guess, it's from `setuptools`) https://github.com/AnacondaRecipes/pyee-feedstock/blob/915ec9a9343305c6fd1268157c531222bb44ecba/recipe/run_test.py#L4

Reply:
the python dependency might be over-specified, but it indicates on reading that it is required ... I would love to avoid the implicit dependency and type it explicit